packages/bun/test/sdk.test.ts

```diff
@@ -1,14 +1,20 @@
-import { expect, test } from 'bun:test';
+import { describe, expect, test } from 'bun:test';
 
 import { init } from '../src/index';
 
-test("calling init shouldn't fail", () => {
-  init({
+describe('Bun SDK', () => {
+  const initOptions = {
     dsn: 'https://00000000000000000000000000000000@o000000.ingest.sentry.io/0000000',
+    tracesSampleRate: 1,
+  };
```

Comment:
It turns out that the SDK initialised in here is effecting the SDK being used by `bunserver.test.ts` above. I am not familiar enough with the Sentry codebase to know how to properly reset after these tests but for sure something is missing here as the tests are not properly isolated.

I've had to add `tracesSampleRate: 1` for now to ensure that the SDK being reused doesn't totally break the other tests which depend on tracing happening for every single call.